lib/note-detail/heading/constants.js

```diff
@@ -0,0 +1,3 @@
+export const headingPrefixRegex = /^([#]{1,6)\s+[^\s]/g;
+
+export const headingRegex = /^([#]{1,6})(\s+)([^\s]+)/g;
```

Comment:
These are specific enough and simple enough that we can keep them in the files where they are used.

Reply:
Addressed in 67db72e49a9c2c5719c035ebdbbe79668296d410.